sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/QueueReceiverClient.cs

```diff
@@ -7,12 +7,12 @@
 using Azure.Core;
 using Azure.Messaging.ServiceBus.Core;
 
-namespace Azure.Messaging.ServiceBus.Receiver
+namespace Azure.Messaging.ServiceBus
```

Comment:
Should these types move to the root folder as part of the namespace change?

Reply:
This will probably change around again so leaving for now.